config/checkstyle-input-suppressions.xml

```diff
@@ -68,6 +68,28 @@
           files="[\\/]packageobjectfactory[\\/]zoo[\\/]FooCheck.java"/>
 
   <!-- until https://github.com/checkstyle/checkstyle/issues/13213 -->
+  <suppress id="UnnecessaryOkComment"
```

Comment:
We need to remove such suppressions.
But you are adding them.

Reply:
Now I got it!! I am working on it @romani .

Reply:
@romani, I have changed files and made PR again. Could you please kindly check it?